src/BetterPhpDocParser/PhpDocParser/DoctrineAnnotationDecorator.php

```diff
@@ -33,10 +33,10 @@
     private const ALLOWED_SHORT_ANNOTATIONS = ['Target'];
 
     /**
-     * @see https://regex101.com/r/95kIw4/2
+     * @see https://regex101.com/r/bGp2V0/1
      * @var string
      */
-    private const LONG_ANNOTATION_REGEX = '#@\\\\(?<class_name>.*?)(?<annotation_content>\(.*?\)|,)#';
+    private const LONG_ANNOTATION_REGEX = '#@\\\\(?<class_name>.*?)(?<annotation_content>\(.*?\)|,|\n|$)#';
```

Comment:
this should support windows new line with `\r?\n`

```suggestion
    private const LONG_ANNOTATION_REGEX = '#@\\\\(?<class_name>.*?)(?<annotation_content>\(.*?\)|,|\r?\n|$)#';
```

Reply:
https://github.com/rectorphp/rector-src/pull/5610